src/components/logs-launcher.tsx

```diff
@@ -40,10 +40,10 @@ export function LogsLauncher({ logs, status }: IProps): JSX.Element {
       >
         {error ? `View Errors` : `View logs`}
       </Button>
-      <Modal isOpen={isOpen}>
-        <ModalPanel position="right" sx={{ width: `100vw` }}>
+      <Modal aria-label="Logs viewer" isOpen={isOpen} onDismiss={toggleLogs}>
```

Comment:
Did you consult with @madalynrose if this is the best solution for a11y here? I just remember her saying aria was a last resort and semantic html is best.

Reply:
I did not, and I should. I based this on the examples in the gatsby-interface Storybook